coralproject/talk
# [CORL-2545] Show singular site options for banning on single site organizations

## What does this PR do?

Update ban modal dialog options for single site organizations so that it shows a single ban/unban option when there is only one site in the organization.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indexes.

## How do I test this PR?

- Setup Coral so there is only one site listed under `Admin > Configure > Organization > Sites`
- Create a mod user and log in as them
- Go to `Community` and attempt to ban a user
- See that the flow is simply to ban the user outright, no `All sites` or `Specific sites` options
- Ban them
- Go to `Manage ban` on same user
- See that the flow is simply to unban the user outright
- Go to `Admin > Configure > Organization > Sites` and add a new site
- Go back to `Community` and attempt to ban user again
- See that the flow now has `All sites` and `Specific sites` options, with save button for ban
- Ban them
- Go to `Manage ban` on the same user
- See that the flow now has `All sites` and `Specific sites` options, with save button for unban
- Unban them
 
## How do we deploy this PR?

No special considerations for deploy.
